torch/_inductor/codegen/mps.py

```diff
@@ -463,6 +463,13 @@ def reduction(
                 f"c10::metal::threadgroup_{reduction_type}({acc_buf}, {reduction_dim.numel})",
                 dtype=dtype,
             )
+        if reduction_type == 'welford_reduce':
+            return self.welford_reduce_fallback(dtype, value)
+            # TODO: This is how it supposed ot be
```

Comment:
typo: "ot" -> "to"

Reply:
or just remove the comment altogether.

Reply:
That's the plan, when I have a proper implementation :)